app/presenters/publishing_api/world_location_news_presenter.rb

```diff
@@ -57,7 +63,11 @@ def featured_links
     end
 
     def path_for_news_page
-      Whitehall.url_maker.world_location_news_index_path(world_location)
+      if world_location.world_location?
+        Whitehall.url_maker.world_location_news_index_path(world_location)
+      elsif world_location.international_delegation?
+        Whitehall.url_maker.world_location_path(world_location, locale: I18n.locale)
```

Comment:
Why don't world location news need this locale in their path too?

Reply:
It looks like I've added in the locale too early (it was supposed to be in [this PR](https://github.com/alphagov/whitehall/pull/6961/commits/ac3fc6487a540539ad3916e5a74f9bc1f366ea25#diff-e30a477531e7255e6c1245b17ca6e74324183262f63e8bf6a960102203fef22cR60) which will come after this).

At the moment, we're using the Rails automagic route helper method, so Rails deals with the locale itself.  After we remove the route in a later PR, we'll need to use our own helper which requires us to specify the locale.

I'll remove this from here as we don't need it yet.